.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -55,8 +69,7 @@ body:
     attributes:
       label: Versions and configuration
       description: |
-        The following information should consist only of the automated game report generated by the game; please put further information below.
-        To generate a report: "Main menu (ESC) > Debug Menu > Info > Generate game report".
+        `F1`Debug Menu -> `i`Info -> `r`Generate game report
```

Comment:
Maybe move the minimized save archive option to the main menu's debug menu? The real debug menu contains way too many options that are not supposed to be used in a normal gameplay.

Reply:
That shows the full debug menu if the key is bound and I didn't realize there was a limited set always available. Thanks!